.editorconfig

```diff
@@ -0,0 +1,19 @@
+# EditorConfig helps developers define and maintain consistent
+# coding styles between different editors and IDEs
+# http://editorconfig.org
+
+root = true
+
+[*]
+indent_style = space
+indent_size = 4
+end_of_line = lf
+charset = utf-8
+trim_trailing_whitespace = true
+insert_final_newline = true
+
+[*.md]
+trim_trailing_whitespace = false
+
+[*.adoc]
+trim_trailing_whitespace = false
```

Comment:
This is better handled with checkstyle-plugin on build / CI to enforce rules.

`.editorconfig` support on Eclipse IDE is an optional plugin (with very few downloads), with limited type support (Java and XML only last time I looked).

The support on IntelliJ is also disabled if you use a "Settings Repository" to manage configuration across multiple projects.

Reply:
In fact, just drop this file entirely.
There's only 1 project on `github.com/eclipse-ee4j` that defines this - https://github.com/eclipse-ee4j/krazo

Krazo is not a Spec / API project, it's an implementation project.

Reply:
I do agree with this, we should add validation using checkstyle.
Can we remove it post-merge?